packages/venia-ui/lib/components/CmsBlock/cmsBlock.js

```diff
@@ -14,7 +14,9 @@ const CmsBlockGroup = props => {
     const classes = mergeClasses(defaultClasses, props.classes);
 
     const { loading, error, data } = useQuery(GET_CMS_BLOCKS, {
-        variables: { identifiers }
+        variables: { identifiers },
+        fetchPolicy: 'cache-and-network',
+        nextFetchPolicy: 'cache-first'
```

Comment:
wouldn't cache-first mean that updated content would not show if it was edited again?
https://medium.com/@galen.corey/understanding-apollo-fetch-policies-705b5ad71980?

Reply:
> wouldn't cache-first mean that updated content would not show if it was edited again?

On first load, cache-and-network _always_ updates with network (server) data. But if that data ever changes on the client, the query makes _another_ call to the server. We applied the `nextFetchPolicy` to avoid excessive server calls as it allows the initial network fetch but then always trusts the cache from then on (for this page load).